src/flamenco/types/fd_types_custom.c

```diff
@@ -172,17 +172,15 @@ void fd_solana_vote_account_decode_unsafe( fd_solana_vote_account_t * self, fd_b
   ulong data_len;
   fd_bincode_uint64_decode_unsafe( &data_len, ctx );
   if( data_len ) {
-    uchar * data = fd_valloc_malloc( ctx->valloc, 8UL, data_len );
```

Comment:
This change was made because we don't want to keep a variable sized solana account in here. Does this change that?

Reply:
I think we need to do a 
```
uchar * data = fd_scratch_alloc(...);
```

Reply:
this was just copying the memory for no purpose. it works just fine without